tests/tacacs/test_ro_user.py

```diff
@@ -16,7 +16,7 @@
 
 def ssh_remote_run(localhost, remote_ip, username, password, cmd):
     res = localhost.shell("sshpass -p {} ssh "\
-                          "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null "\
+                          "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR "\
```

Comment:
>LogLevel [](http://example.com/codeflow?start=90&length=8)

Could you add comment to explain this change?

Reply:
Change by mistake, reverted.